backend/ordd_api/views.py

```diff
@@ -32,7 +32,7 @@
     DatasetListSerializer, DatasetPutSerializer, DatasetsDumpSerializer)
 from .models import (Region, Country, OptIn, Dataset, KeyDataset,
                      KeyDatasetName, KeyCategory, KeyTag,
-                     my_random_key)
+                     my_random_key, Profile)
 from .mailer import mailer
 from ordd_api import __version__, MAIL_SUBJECT_PREFIX
 from ordd.settings import (ORDD_ADMIN_MAIL, EMAIL_CONFIRM_PROTO)
```

Comment:
Isn't `ORDD_ADMIN_MAIL` used anymore? Yes that's teh case please remove it form imports